src/cloud-api-adaptor/install/README.md

```diff
@@ -1,144 +1,126 @@
-## Installation
+# Installation
 
-* **Setup Cloud Resources**
+## Prerequisites
 
-  If using AWS, create VPC and AMI. Similarly for other providers create the
-  necessary resources.
+- validate kubectl is available in your `$PATH` and `$KUBECONFIG` is set to point to your Kubernetes cluster
+- `yq` tool is available in your `$PATH`
+- At least one node in the cluster must have the "worker" role.
 
-* **Setup Kubernetes cluster in the cloud**
-
-  At least one node in the cluster must have the "worker" role.
   Verify by executing the following command.
+
+  ```sh
+  kubectl get nodes
   ```
-   kubectl get nodes
-  ```
+
   You should see "worker" under the "ROLES" column as shown below:
-  ```
-   NAME             STATUS   ROLES                         AGE   VERSION
-   testk-master-0   Ready    control-plane,master,worker   37h   v1.25.0
+  
+  ```sh
+  NAME             STATUS   ROLES                         AGE   VERSION
+  testk-master-0   Ready    control-plane,master,worker   37h   v1.25.0
   ```
 
   If "worker" role is missing, execute the following command to set the role.
 
-    ```
-    export NODENAME=<node-name>
-    kubectl label node $NODENAME node.kubernetes.io/worker=
-    ```
-
-## Deploy webhook
-
-   Please refer to the instructions available in the following [doc](../../webhook/docs/INSTALL.md).
+  ```sh
+  export NODENAME=<node-name>
+  kubectl label node $NODENAME node.kubernetes.io/worker=
+  ```
 
-## Configure and deploy CoCo and the cloud-api-adaptor
+## Deploy CoCo operator and cloud-api-adaptor daemonset
 
 - Update the `kustomization.yaml` file in `install/overlays/$(CLOUD_PROVIDER)/kustomization.yaml` with your own settings
 - Optionally [set up authenticated registry support](../docs/registries-authentication.md)
+- Install
 
-## Deploy the CoCo operator, CC runtime CRD and the cloud-api-adaptor daemonset
-
-### Using make deploy
-
-You can deploy the CoCo operator and cloud-api-adaptor with the `Makefile` by running
-* set CLOUD_PROVIDER
-    ```
-    export CLOUD_PROVIDER=<aws|azure|ibmcloud|ibmcloud-powervs|libvirt|vsphere>
-    ```
-    * `RESOURCE_CTRL` is set to `true` by default to allow the peerpod-ctrl to run, monitor and delete dangling cloud resources
+  ```sh
+  export CLOUD_PROVIDER=<aws|azure|ibmcloud|ibmcloud-powervs|libvirt|vsphere>
+  make deploy
+  ```
 
-* `make deploy` deploys operator, runtime and cloud-api-adaptor pod in the configured cluster
-    * validate kubectl is available in your `$PATH` and `$KUBECONFIG` is set
-    * `yq` tool is available in your `$PATH`
+  This will deploy the latest version.
```

Comment:
nit: latest release? if yes maybe "release" is a bit less confusing

Reply:
sure..